src/RuleSet/Sets/PSR12Set.php

```diff
@@ -29,7 +29,10 @@ public function getRules(): array
             'braces' => [
                 'allow_single_line_anonymous_class_with_empty_body' => true,
             ],
-            'class_definition' => ['space_before_parenthesis' => true], // defined in PSR12 ¶8. Anonymous Classes
+            'class_definition' => [
+                'fix_constructor_arguments' => false, // handled by method_argument_space fixer
+                'space_before_parenthesis' => true, // defined in PSR12 ¶8. Anonymous Classes
```

Comment:
what is ` ¶8`?

Reply:
I think it's supposed to mean “section 8”. From what I understand, it should be `§ 8`, as pilcrow (`¶`) means a paragraph, and what we want is section sign (`§`). [This](https://en.wikipedia.org/wiki/Section_sign#Use) section of Wikipedia article contains some information about the confusion of section and paragraph.

Anyway, keep in mind that it was there before and is not my addition.

Reply:
@jrmajor you right